### DISCUSSION TITLE
How to unistall or remove security onion from centos

### DISCUSSION THREAD

**Discussion by abuyazeen** [Security-Onion-Solutions/securityonion#8181] [Unsupported Versions] [answered]
Hi , i want uninstall or remove old version security onion from my vm centos , and install new one any tips

**Comment by reyesj2** [comment]
If your goal is to go from an older version to newest version why not use soup? 
https://docs.securityonion.net/en/2.3/soup.html?highlight=soup#soup

You could also just get rid of the old VM, create a new VM, and point it to the latest 2.3.130 ISO, and install a fresh build
https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md

**Comment by TOoSmOotH** [comment, marked as answer]
If you run setup again it will write over your current config. There is no need to uninstall.
